src/test/kotlin/blackjack/domain/card/DeckTest.kt

```diff
@@ -0,0 +1,30 @@
+package blackjack.domain.card
+
+import blackjack.domain.card.Rank.ACE
+import blackjack.fixtures.createCard
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class DeckTest : BehaviorSpec({
+    Given("덱에 카드가 존재하는 경우") {
+        val deck = Deck(mutableListOf(createCard("A")))
+
+        When("드로우하면") {
+            Then("카드를 반환한다") {
+                deck.draw().rank shouldBe ACE
```

Comment:
```suggestion
            val card =  deck.draw()
            Then("카드를 반환한다") {
               card.rank shouldBe ACE
```
실행부와 검증부와 명확하게 나누면, 
가독성 측면이나, 무엇을 테스트해야하는지 좀더 명확히할수 있을거 같아요 :)

Reply:
넵 테스트에 명확성을 부여하도록 하겠습니다 🙇‍♂️